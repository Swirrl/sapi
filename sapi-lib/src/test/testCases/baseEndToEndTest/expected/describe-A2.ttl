@prefix eg:    <http://localhost/example/> .
@prefix dct:   <http://purl.org/dc/terms/> .
@prefix rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@prefix owl:   <http://www.w3.org/2002/07/owl#> .
@prefix xsd:   <http://www.w3.org/2001/XMLSchema#> .
@prefix skos:  <http://www.w3.org/2004/02/skos/core#> .
@prefix rdfs:  <http://www.w3.org/2000/01/rdf-schema#> .
@prefix jtext: <http://jena.apache.org/text#> .
@prefix foaf:  <http://xmlns.com/foaf/0.1/> .

<http://localhost/flood-monitoring/example/A2>
        a              <http://localhost/flood-monitoring/example/Thing> ;
        rdfs:label     "A2" ;
        <http://localhost/flood-monitoring/example/group>
                "A" ;
        skos:narrower  <http://localhost/flood-monitoring/example/B2> ;
        skos:notation  2 .

[ rdfs:comment       "WARNING: Pre-alpha test service, data may not be up to date" ;
  rdfs:seeAlso       <http://localhost/flood-monitoring/doc/api-documentation.html> ;
  dct:hasFormat      <http://localhost/flood-monitoring/example/A2.json> , <http://localhost/flood-monitoring/example/A2.rdf> ;
  dct:license        "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/" ;
  dct:publisher      "Epimorphics.com" ;
  owl:versionInfo    "0.1" ;
  foaf:primaryTopic  <http://localhost/flood-monitoring/example/A2>
] .
