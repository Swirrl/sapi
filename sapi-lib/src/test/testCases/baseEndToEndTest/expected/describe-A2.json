{ 
  "@context" : "http://localhost/flood-monitoring/doc/context-TBD.jsonld" ,
  "meta" : { 
    "publisher" : "Epimorphics.com" ,
    "licence" : "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/" ,
    "documentation" : "http://localhost/flood-monitoring/doc/api-documentation.html" ,
    "version" : "0.1" ,
    "comment" : "WARNING: Pre-alpha test service, data may not be up to date" ,
    "hasFormat" : [ "http://localhost/flood-monitoring/example/A2.csv", "http://localhost/flood-monitoring/example/A2.rdf", "http://localhost/flood-monitoring/example/A2.ttl" ]
  }
   ,
  "items" : [ { 
    "@id" : "http://localhost/flood-monitoring/example/A2" ,
    "group" : "A" ,
    "label" : "A2" ,
    "narrower" : { 
      "@id" : "http://localhost/flood-monitoring/example/B2"
    }
     ,
    "notation" : 2 ,
    "type" : { 
      "@id" : "http://localhost/flood-monitoring/example/Thing"
    }
  }
   ]
}
